Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5334,34 +5653,137 @@ def panorama_install_latest_content_update(target: str):
+    cmd = demisto.command()
+    script_results = []
+
+    # Is polling requested
+    if use_polling:
+        ScheduledCommand.raise_error_if_not_supported()
+
+        # Create new job if this is not an existing polling job
+        if not job_id:
+
+            # Create a new installation schedule
+            result = panorama_install_latest_content_update(target)
+            if result['response']['@status'] == 'error':
+                if 'msg' in result['response'] and 'line' in result['response']['msg']:
+                    message = '. Reason is: ' + result['response']['msg']['line']
+                    raise Exception('Update content failed' + message)
+                else:
+                    raise Exception('Update content failed.')
 
-    if 'result' in result['response']:
-        # installation has been given a jobid
-        content_install_info = {
-            'JobID': result['response']['result']['job'],
-            'Status': 'Pending'
-        }
-        entry_context = {"Panorama.Content.Install(val.JobID == obj.JobID)": content_install_info}
-        human_readable = tableToMarkdown('Result:', content_install_info, ['JobID', 'Status'], removeNull=True)
+            if 'response' not in result or 'result' not in result['response'] or 'job' not in result['response']['result']:
+                raise Exception('Missing JobID in response.')
 
-        return_results({
-            'Type': entryTypes['note'],
-            'ContentsFormat': formats['json'],
-            'Contents': result,
-            'ReadableContentsFormat': formats['markdown'],
-            'HumanReadable': human_readable,
-            'EntryContext': entry_context
-        })
+            job_id = result['response']['result']['job']
+
+            polling_args = {
+                "job_id": job_id,
+                **args
+            }
+
+            scheduled_command = ScheduledCommand(
+                command=cmd,
+                next_run_in_seconds=interval_in_seconds,
+                args=polling_args,
+                timeout_in_seconds=timeout
+            )
+
+            readable_output = f"Started instllation of latest content update (Job ID: {job_id})"
+            script_results.append(CommandResults(
+                readable_output=readable_output,
+                scheduled_command=scheduled_command))
+
+        # Check the existing job
+        else:
+            result = panorama_content_update_install_status(target, job_id)
+            if result['response']['@status'] == 'error':
+                if 'msg' in result['response'] and 'line' in result['response']['msg']:
+                    message = '. Reason is: ' + result['response']['msg']['line']
+                    raise Exception('Update content failed' + message)
+                else:
+                    raise Exception('Update content failed.')
+
+            if 'response' not in result or 'result' not in result['response'] or 'job' not in result['response']['result']:
```

Comment:
Do we need to fail out here? Does it not being returned mean some error happened?
 Dont we already know the jobid? Maybe add a Warn log here?

Reply:
There is no reason to raise an exception here. I'll change to a warning and continue with the polling.